transport/logging.go

```diff
@@ -48,7 +48,12 @@ func LoggingDialer(d Dialer, logger *logp.Logger) Dialer {
 func (l *loggingConn) Read(b []byte) (int, error) {
 	n, err := l.Conn.Read(b)
 	if err != nil && !errors.Is(err, io.EOF) {
-		l.logger.Debugf("Error reading from connection: %v", err)
+		// Check for a closed network connection error
+		if errors.Is(err, net.ErrClosed) {
+			l.logger.Debugf("Connection is closed: %v", err)
```

Comment:
I think the attached issue mentions that we shouldn't log the error in cases of ErrClosed. But, you're still logging it, with a different message.

Reply:
I think the issue description was a bit misleading, please read my comment here https://github.com/elastic/beats/issues/40824#issuecomment-2431409665